### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elbebass**
* Use capital letter in domain name

### CODE REVIEW

**Review by oltmanns-leuchtfeuer — APPROVED** [review]
LGTM!

**Review by MadlenF — COMMENTED** [review]
why would you change that?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Auth0 for Mautic
 ================
-![Auth0Mautic](https://www.leuchtfeuer.com/fileadmin/github/auth0-for-mautic/Mautic-Auth0.png "Auth0 for Mautic")
+![Auth0Mautic](https://www.Leuchtfeuer.com/fileadmin/github/auth0-for-mautic/Mautic-Auth0.png "Auth0 for Mautic")
```
Comment (oltmanns-leuchtfeuer) [inline README.md:3]: Please use lower capital "L".

**Review by oltmanns-leuchtfeuer — CHANGES_REQUESTED** [review]
Please use lower capital "L"

**Pull request closed without merging**